portindex2json/portindex2json.tcl

```diff
@@ -24,8 +96,13 @@ while {[gets $fd line] >= 0} {
     array set portinfo $line
     array unset json_portinfo
     foreach key [array names portinfo] {
-        if { $key == "categories"} {
-            set json_portinfo($key) [::json::write array {*}$portinfo($key)]
+        if { $key == "categories"  || $key=="variants" || $key=="depends_lib" || $key=="depends_build" || $key=="depends_fetch" } {
+            set json_portinfo($key) [::json::write array {*}[format_strings $portinfo($key)]]
+        } elseif { $key == "maintainers"} {
+            set json_portinfo($key) [parse_maintainers $portinfo($key)]
+            set json_portinfo(closedmaintainer) [is_closedmaintainer $portinfo($key)]
```

Comment:
Not a complaint about this particular line, but I would like to see some further cosmetic changes to both of these functions, but I need some further thinking. I'm sending feedback about the rest now.